java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -117,6 +117,7 @@ open class StorageService : ResurrectorService() {
 
     override fun onDestroy() {
         super.onDestroy()
+        WorkManager.getInstance(this).cancelUniqueWork(PeriodicCleanupTask.WORKER_TAG)
```

Comment:
I don't think we want this here.. we still want to TTL things even if the service was shut down (we want the work manager to wake it back up)

Reply:
Maybe given PeriodicCleanupTask a shutdown signal, and have it cancel itself after the next time it runs?

Reply:
I removed it for now as the periodic task no longer holds the database open. I'll refactor it to allow cancellation in a separate PR.